cities/bad-homburg.json

```diff
@@ -0,0 +1,48 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                  8.61695,
+                  50.22670
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Bad Homburger Wochenmarkt",
+                "location": "Zwischen Markt- und Kurhausvorplatz, Bad Homburg",
+                "opening_hours": "Tu,Fr 07:30-14:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.61524,
+            	    50.22730
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Bad Homburger Blumenmarkt",
+                "location": "Waisenhausplatz, Bad Homburg",
+                "opening_hours": "Sa 09:00-16:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Bad Homburg ver der Höhe",
+            "url": "http://www.bad-homburg.de/leben-in-bad-homburg/sport-freizeit-ehrenamt/Wochen-_und_Blumenmarkt.php"
+        },
+        "map_initialization": {
+            "coordinates": [
+                8.615324,
+	      	    50.227428
+            ],
+            "zoom_level": 12
```

Comment:
The two markets appear very close on the map both on mobile and a desktop browser. Do you want to change the zoom level so they can be distinguished easier?

Further, please put all your changes in one commit on your branch since the later commits are just fixes. No need to keep them in the history.